docs/Local_Development_Tutorial.md

```diff
@@ -0,0 +1,168 @@
+# Local Development Tutorial: Getting Started with MarkLogic in Kubernetes
+
+# Table of contents
+* [Introduction](#Introduction)
+* [Prerequisites](##Prerequisites)
+* [Procedure](#Procedure)
+* [Setting Up Minikube](##Setting-Up-Minikube)
+* [Installing MarkLogic to Minikube](##Installing-a-Single-MarkLogic-Host-to-Minikube)
+* [Installing Multiple MarkLogic Hosts to Minikube](##Installing-Multiple-MarkLogic-Hosts-to-Minikube)
+* [Verifying the Installation](##Verifying-the-Installation)
+* [Debugging](#Debugging)
+* [Cleanup](#Cleanup)
+
+# Introduction
+This tutorial describes how to set up local Kubernetes development environment with Minikube and MarkLogic Server. It covers these tasks:
+- Set up the prerequisites necessary for local development using MarkLogic Server in Kubernetes 
+- How to run Minikube and load MarkLogic Server into the Kubernetes cluster 
+- Access the MarkLogic Server cluster
+- How to debug the Kubernetes environment 
+- How to clean up your environment
+
+
+## Prerequisites
+The following steps assume you are running this tutorial from a desktop environment. Mobile environments will likely experience problems and may not work.
+- [Docker](https://docs.docker.com/engine/install/) Subscribe to Docker Hub and pulldown the latest image from: https://hub.docker.com/_/marklogic
```

Comment:
Add a colon after Docker to separate it from the rest of the sentence.

Reply:
Updated 51b4b8e